docs/flexbox.md

```diff
@@ -9,10 +9,37 @@ You will normally use a combination of `flexDirection`, `alignItems`, and `justi
 
 > Flexbox works the same way in React Native as it does in CSS on the web, with a few exceptions. The defaults are different, with `flexDirection` defaulting to `column` instead of `row`, and the `flex` parameter only supporting a single number.
 
+### Layout Direction
```

Comment:
What do you think about moving this section down? It may flow better if something like `flex` or `flex direction` is first